package.json

```diff
@@ -100,7 +100,7 @@
     "typescript": "~5.3.3"
   },
   "peerDependencies": {
-    "expensify-common": ">=2.0.106",
+    "expensify-common": ">=2.0.108",
```

Comment:
No need to bump this in my opinion, but if you do bump this, please also update README.md accordingly.